app/views/schools/choose_programme/success.html.erb

```diff
@@ -2,10 +2,75 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= govuk_panel(
-      title_text: "Induction programme confirmed",
-      text: "You can now view the next steps for your programme.",
-      classes: "govuk-!-margin-bottom-8") %>
+      title_text: "Training programme confirmed",
+      classes: "govuk-!-margin-bottom-7") %>
+
+    <%- if school_cohort.school_chose_fip? -%>
```

Comment:
Do you have any tests for these conditions?

Reply:
For that condition the spec `Manage FIP training FIP Induction Coordinator with training provider` was failing and I had to change the copy it was testing.
I suppose the last 2 conditions (other than CIP and FIP) are not covered, but I'm not sure which steps they involve, or if there's a way at all to get there, I kinda figured out myself the changes to that page would involve triggering those attributes, and definitely needs to be double checked by someone more knowledgeable on the software.

Reply:
I think it's worth taking the time to figure out how to write tests for this. The induction programme choice a school makes is an important part of the journey that Register & Partner are responsible for. 

Very happy to jump on a call to discuss anything you're not sure about, or if you prefer, ask in Slack!